contrib/README.md

```diff
@@ -14,7 +14,7 @@ ansible-playbook ovn-kubernetes-cluster.yml
 ```
 
 Currently supported Linux nodes:
-- Ubuntu 14.04 and 16.04
+- Ubuntu 18.04 and 16.04
```

Comment:
I have been using this Ansible playbook on 18.04 and was able to setup a working cluster. I did not notice any compile errors. At which step in the playbook did you get the compile error?

Reply:
I take back my claim. It seems there were indeed some problems that I did not notice at first. I did not see any compile errors from OVS, but it seems that there were problems running the KubeDNS containers. For the time being I have reverted my VMs to 16.04.

Reply:
Sorry I haven't posted this sooner:
```
sudo apt install ../openvswitch-datapath-dkms_2.9.2-1_all.deb
Reading package lists... Done
Building dependency tree
Reading state information... Done
Note, selecting 'openvswitch-datapath-dkms' instead of '../openvswitch-datapath-dkms_2.9.2-1_all.deb'
The following package was automatically installed and is no longer required:
  linux-headers-4.15.0-22
Use 'sudo apt autoremove' to remove it.
The following NEW packages will be installed:
  openvswitch-datapath-dkms
0 upgraded, 1 newly installed, 0 to remove and 161 not upgraded.
Need to get 0 B/4,835 kB of archives.
After this operation, 52.1 MB of additional disk space will be used.
Get:1 /home/ubuntu/openvswitch-datapath-dkms_2.9.2-1_all.deb openvswitch-datapath-dkms all 2.9.2-1 [4,835 kB]
Selecting previously unselected package openvswitch-datapath-dkms.
(Reading database ... 134217 files and directories currently installed.)
Preparing to unpack .../openvswitch-datapath-dkms_2.9.2-1_all.deb ...
Unpacking openvswitch-datapath-dkms (2.9.2-1) ...
Setting up openvswitch-datapath-dkms (2.9.2-1) ...

Creating symlink /var/lib/dkms/openvswitch/2.9.2/source ->
                 /usr/src/openvswitch-2.9.2

DKMS: add completed.

Kernel preparation unnecessary for this kernel.  Skipping...

Building module:
cleaning build area...(bad exit status: 2)
./configure --with-linux='/lib/modules/4.15.0-39-generic/build' && make -C datapath/linux.........(bad exit status: 2)
ERROR: Cannot create report: [Errno 17] File exists: '/var/crash/openvswitch-datapath-dkms.0.crash'
Error! Bad return status for module build on kernel: 4.15.0-39-generic (x86_64)
Consult /var/lib/dkms/openvswitch/2.9.2/build/make.log for more information.

cat /etc/lsb-release
DISTRIB_ID=Ubuntu
DISTRIB_RELEASE=18.04
DISTRIB_CODENAME=bionic
DISTRIB_DESCRIPTION="Ubuntu 18.04 LTS"
```

I think it's fine if you change:
`Ubuntu 18.04 and 16.04` to 
`Ubuntu 16.04 and 18.04 (experimental)`